### PR TITLE
Reset `POT-Creation-Date` in translation file to avoid git conflicts

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

I noticed that #1821 wasn't quite enough to reduce the git conflicts in the translation file due to the constantly changing date...
Since this information isn't really that useful anyway, I suggest to remove it in our dev tools.

### Proposed changes

- Reset `POT-Creation-Date` in translation file to avoid git conflicts

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by JoeyStk — APPROVED** [review]
Thank you a lot! This is a very helpful PR :) 
I'm absolutely not a bash expert but as far as I can tell this looks good :) Maybe it would be good if someone with more experience in bash could also take a look at this :)

**Comment by timoludwig** [comment]
> Maybe it would be good if someone with more experience could also take a look at this :)

@svenseeberg I can picture you really wanting to read some Bash :joy:

File: dev-tools/translate.sh [hunk 1]
```diff
@@ -12,17 +12,21 @@ ensure_not_root
...
-fi
+# Reset POT-Creation-Date to avoid git conflicts
+sed --in-place --regexp-extended 's/^"POT-Creation-Date: [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}\+[0-9]{4}\\n"$/"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\\n"/' "${TRANSLATION_FILE}"
 
-# Compile translation file
-echo "Compiling translation file..." | print_info
-pipenv run integreat-cms-cli compilemessages --verbosity "${SCRIPT_VERBOSITY}"
+# Skip compilation if --skip-compile option is given
+if [[ "$*" == *"--skip-compile"* ]]; then
+    # Compile translation file
```
Comment (charludo) [inline dev-tools/translate.sh:27]:
```suggestion
# Skip compilation if --skip-compile option is given
if [[ "$*" != *"--skip-compile"* ]]; then
    # Compile translation file
```
😉

**Review by charludo — CHANGES_REQUESTED** [review]
Agree, really useful change!! Other than the one comment, LGTM

File: dev-tools/translate.sh (hunk 1 above)
Comment (timoludwig) [reply dev-tools/translate.sh:27]: oh haha :sweat_smile:

Comment (david-venhoff) [reply dev-tools/translate.sh:27]: Seems to still be `==`

Comment (timoludwig) [reply dev-tools/translate.sh]: :facepalm:

**Review by david-venhoff — APPROVED** [review]
Looks good to me now :tada:

**Pull request merged**